pkg/cmd/kafka/create/create.go

```diff
@@ -53,6 +52,7 @@ type Options struct {
 	Connection factory.ConnectionFunc
 	Logger     func() (logging.Logger, error)
 	localizer  localize.Localizer
+	validator  *pkgKafka.Validator
```

Comment:
Hmm, i would say there is no need to add the localizer to `opts` - just initialize it when you need it.